### PR TITLE
Added more Micronaut projects (aws, data, kafka, kotlin, security, sp…

### PR DESCRIPTION
**Pull request by ribafish**
Added more Micronaut projects: aws, data, kafka, kotlin, security, spring, starter.

Based the runs on `.github/workflows/gradle.yml` files in respective repos:

 - [aws](https://github.com/micronaut-projects/micronaut-aws/blob/master/.github/workflows/gradle.yml)
 - [data](https://github.com/micronaut-projects/micronaut-data/blob/master/.github/workflows/gradle.yml)
 - [kafka](https://github.com/micronaut-projects/micronaut-kafka/blob/master/.github/workflows/gradle.yml)
 - [kotlin](https://github.com/micronaut-projects/micronaut-kotlin/blob/master/.github/workflows/gradle.yml)
 - [security](https://github.com/micronaut-projects/micronaut-security/blob/master/.github/workflows/gradle.yml)
 - [spring](https://github.com/micronaut-projects/micronaut-spring/blob/master/.github/workflows/gradle.yml)
 - [strarter](https://github.com/micronaut-projects/micronaut-starter/blob/4.4.x/.github/workflows/gradle.yml)

### CODE REVIEW

File: .github/workflows/run-experiments-micronaut-core.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ env:
   GRADLE_ENTERPRISE_URL: "https://ge.solutions-team.gradle.com"
   GIT_REPO: "https://github.com/micronaut-projects/micronaut-core"
   TASKS: "check"
-  ARGS: "-x :http-client:test -x :inject:test -x japiCmp"
+  ARGS: "-x :http-client:test -x :inject:test -x japiCmp -DpredictiveTestSelection=false"
```
Comment (runningcode) [inline .github/workflows/run-experiments-micronaut-core.yml:14]: how come for some we use an argument and for others it is an environment variable?

**Review by runningcode — COMMENTED** [review]
looks good, just one question.

File: .github/workflows/run-experiments-micronaut-core.yml (hunk 1 above)
Comment (ribafish) [reply .github/workflows/run-experiments-micronaut-core.yml:14]: For which workflows is it an env var? As far as I can see it's always an argument...

Comment (runningcode) [reply .github/workflows/run-experiments-micronaut-core.yml:14]: I mean a Github environment variable. Like the other ones in this PR, like this one: https://github.com/gradle/develocity-oss-projects/pull/99/files#diff-1afdc0ab7b2a93c50260767c5d37da48249a6ffdbefc0c1c431d0209d3e54ac0R14

Comment (ribafish) [reply .github/workflows/run-experiments-micronaut-core.yml:14]: I'm lost, both are using the `ARGS` github env var.

Comment (runningcode) [reply .github/workflows/run-experiments-micronaut-core.yml:14]: You are right, I'm confused here. In that case the PR is approved.

**Review by runningcode — APPROVED** [review]

**Pull request merged**